core/src/main/res/values/colors.xml

```diff
@@ -12,10 +12,13 @@
   <color name="blue800">#1565c0</color>
   <color name="pink300">#f06292</color>
   <color name="actionModeBackground">#4285F4</color>
-  <color name="gray500">#9E9E9E</color>
-  <color name="gray700">#616161</color>
-  <color name="text_secondary">@color/gray700</color>
-  <color name="text_tertiary">@color/gray500</color>
+  <color name="grey350">#FFD6D6D6</color>
```

Comment:
I don't care too much about `gray` vs `grey` but the american `gray` is technically more correct. I hate it less than spelling color without a `u` anyhow.

Keep it `gray`.